From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Dreeam <61569423+Dreeam-qwq@users.noreply.github.com>
Date: Sun, 30 Apr 2023 06:22:08 -0400
Subject: [PATCH] Fix: Make log4j compatible with future release

Removed since 1.20.6

This patch fixes the warnning "WARN StatusConsoleListener The use of package scanning to locate plugins is deprecated and will be removed in a future release" during server launching.

diff --git a/src/main/resources/log4j2.xml b/src/main/resources/log4j2.xml
index 0769f5c4711a3b7f59489e611ed01ad8367e5db1..a056aa167887abef9e6d531a9edd2cda433567d2 100644
--- a/src/main/resources/log4j2.xml
+++ b/src/main/resources/log4j2.xml
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<Configuration status="WARN" packages="com.mojang.util" shutdownHook="disable">
+<Configuration status="WARN" shutdownHook="disable">
     <Appenders>
         <Queue name="ServerGuiConsole">
             <!-- Purpur start - copied from TerminalConsole -->
